src/profile/ChangePassword.js

```diff
@@ -29,39 +27,42 @@ const ChangePassword = (props) => {
   };
 
   const handleChange = (event) => {
-    setstate({
+    setState({
       ...state,
-      [event.target.id]: event.target.value
+      [event.target.id]: event.target.value,
     });
   };
 
   const handleChangeClick = async (event) => {
     event.preventDefault();
-    setstate({
+    setState({
       ...state,
-      isChanging: true
+      isChanging: true,
     });
 
     try {
       const currentUser = await Auth.currentAuthenticatedUser();
-      await Auth.changePassword(currentUser,state.oldPassword,state.password);
+      await Auth.changePassword(
+        currentUser, 
+        state.oldPassword, 
+        state.password
+      );
       props.history.push("/settings");
```

Comment:
I'm sorry I missed this earlier, but can you please change this from '/settings' to '/' ? The 'settings' page no longer exists, and when we successfully change the password, it takes us to the 'NotFound' route. The component functionally works though, I just cloned and tested it.

Reply:
Doubt : On line 50 instead of using the history object via props why cant we use the useHistory hook there ?